script/wraith.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
```

Comment:
Note to self: will almost certainly not work on Docker because the runner container can't run a wraith container, needs an alternate approach.

Reply:
So, https://bbc-news.github.io/wraith/os-install.html links to a thing for Docker behavior but it seems to give a 404 now. The wraith GitHub itself has a Dockerfile, though, which we could use for this!

https://github.com/BBC-News/wraith/blob/4f6eb0b7625d468bb9e3061fd2628f48df4c4e37/Dockerfile

Reply:
I guess maybe we need imagemagik installed but that doesn't seem to be the thing I was gesturing at? we need to run wraith in a container which is also running localhost at the same time, or they need to be running in separate containers which can talk to each other. I don't think the issue is getting wraith running, it's the coordinating with localhost part that's hard?

Reply:
yeah Throne's thing is a seperate problem. I tried to modify this script so it didn't need to use rails runner but I think I... failed... at that